README.md

```diff
@@ -492,6 +493,12 @@ API | Description | Auth | HTTPS | CORS |
 | [Represent by Open North](https://represent.opennorth.ca/) | Find Canadian Government Representatives | No | Yes | Unknown |
 | [USAspending.gov](https://api.usaspending.gov/) | US federal spending data | No | Yes | Unknown |
 
+**[⬆ Back to Index](#index)**
+### Education
+API | Description | Auth | HTTPS | CORS |
+|---|---|---|---|---|
+| [CollegeScoreCard.ed.gov](https://api.data.gov/ed/collegescorecard/v1/schools) | College Score Card | Yes | Yes | Unknown |
```

Comment:
This fits better for the `Open Data` section. Can you please move it there?

Creating a new section requires having a least 3 records there.